tock/employees/models.py

```diff
@@ -38,3 +39,13 @@ class Meta:
 
     def __str__(self):
         return '%s' % (self.user)
+
+    def save(self, *args, **kwargs):
+        if self.current_employee is False:
+            try:
+                token = Token.objects.get(user=self.user)
+                token.delete()
+            except:
```

Comment:
In general, it's better to do specific exceptions like `except.DoesNotExist(): pass`, so that it's not silently failing on exceptions other than the ones we're expecting. When working on this, were you mostly seeing DoesNotExist, or was it throwing other exceptions as well?